external/fv3fit/tests/test_xr_prediction.py

```diff
@@ -131,3 +141,25 @@ def test_DatasetPredictor_dump_load(tmpdir):
         np.testing.assert_allclose(
             loaded_predictor.predict(in_xr)["out0"], in_xr["in0"]
         )
+
+
+@unittest.skipIf(not cupy_available, "cupy not available")
```

Comment:
Since we don't have CI w/ GPUs set up, could you paste a local test passing of this on Perlmutter?  

We could try setting up a github action that connects to our cluster as in the `full-model` repo and add a mark for `gpu-only` to run GPU-specific tests.  I think that would involve some additional dependency installation pathways for automated env setup for with GPU-specific modules (cupy and cupy_xarray).  I'm fine doing this in a separate PR though.

Reply:
Good idea. Opening a separate issue for enabling GPU-specific test.